microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots.md

```diff
@@ -61,7 +63,8 @@ Choose **Clear all profiles** icon to remove snapshots (both from DevTools and a
 Closing the DevTools window does not delete profiles from the memory associated with the renderer process.  When reopening DevTools, all previously taken snapshots reappear in the list of snapshots.
 
 > [!NOTE]
-> Try out this example of [scattered objects](https://microsoft-edge-chromium-devtools.glitch.me/static/memory/example-03.html) and profile it using the Heap Profiler.  A number of (object) item allocations are displayed.
+> Try out this example of [scattered objects](https://microsoftedge.github.io/Demos/devtools-memory-heap-snapshot/example-03.html) and profile it using the Heap Profiler.  A number of (object) item allocations are displayed.  [Source files for the "Record heap snapshots" demo page](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-memory-heap-snapshot).
```

Comment:
Comment out "[Source files for the "Record heap snapshots" demo page](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-memory-heap-snapshot)" here?  That link is for the whole set, not just example-03.